study.md

```diff
@@ -246,5 +254,8 @@ Pick a few design heuristics. Describe some common, named refactorings you can
 use approach a good design.
 
 ```md
-<!-- your answer here -->
+Is it DRY? This is an essenital design heuristic that simply means Don't Repeat Yourself. If you have already defined a method to do a certain task, for example, never have that task or a very similiar one later in your code.
+The more methods the better. If you can write a smaller method to do a specific task you should do so. This will make your code clearer and easier to read.
```

Comment:
Great start. What would you use to DRY up duplicated logic? Maybe "Extract Method". Have a look at some named refactorings, like "Extract Method", to level up.